# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openexr/gcc43-add-string-header.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- openexr-1.6.1/exrenvmap/main.cpp.orig	2010-07-23 13:49:22.256507913 +0200
+++ openexr-1.6.1/exrenvmap/main.cpp	2010-07-23 13:50:13.948007519 +0200
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <exception>
 #include <stdlib.h>
+#include <string.h>
 
 using namespace Imf;
 using namespace std;
--- openexr-1.6.1/exrmaketiled/main.cpp.orig	2010-07-23 13:49:44.216007533 +0200
+++ openexr-1.6.1/exrmaketiled/main.cpp	2010-07-23 13:50:59.152007675 +0200
@@ -46,6 +46,7 @@
 #include <exception>
 #include <string>
 #include <stdlib.h>
+#include <string.h>
 
 using namespace Imf;
 using namespace std;
